metadata.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
What is this file needed for?

Reply:
My bad, that one snuck it's way in. I'll remove it. Just a sec